lib/node_modules/@stdlib/stats/base/dsnanmean/README.md

```diff
@@ -181,6 +181,107 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dsnanmean.h"
+```
+
+#### stdlib_strided_dsnanmean( N, \*X, strideX )
+
+Calculate the Arithmetic Mean value of a double-precision floating-point strided array, ignoring `NaN` values.
+
+```c
+const float x[] = { 1.0f, 2.0f, 0.0f/0.0f, 3.0f, 0.0f/0.0f, 4.0f, 5.0f, 6.0f, 0.0f/0.0f, 7.0f, 8.0f, 0.0f/0.0f };
+
+double v = stdlib_strided_dsnanmean( 5, x, 2 );
+// returns 1.25
+```
+
+The function accepts the following arguments:
+
+-   **N**: `[in] CBLAS_INT` number of indexed elements.
+-   **X**: `[in] float*` input array.
+-   **strideX**: `[in] CBLAS_INT` stride length for `X`.
+
+```c
+double stdlib_strided_dsnanmean( const CBLAS_INT N, const float *X, const CBLAS_INT strideX );
+```
+
+#### stdlib_strided_dsnanmean_ndarray( N, \*X, strideX, offsetX )
+
+Computes the Arithmetic Mean value of a double-precision floating-point strided array, ignoring `NaN` values and using alternative indexing semantics.
```

Comment:
Same comment. I am also not sure why "arithmetic mean" is capitalized.

Reply:
The description is also incorrect. `X` is not double-precision, but single-precision.